scripts/data-retention-remove-user-data.py

```diff
@@ -52,6 +53,11 @@
         base_url=get_api_endpoint_from_stage(stage),
         auth_token=get_auth_token('api', stage)
     )
+    supplier_frameworks = SupplierFrameworkMethods(
+        api_client=data_api_client,
+        logger=logger,
+        dry_run=dry_run
+    )
```

Comment:
I'm not sure about this class/module name.

While our scripts repo isn't the tidiest, the file name structure is actually relatively consistent:
```
scripts/action-that-script-is-carrying-out.py    # top level script, parsing args, initialising api clients etc
dmscripts/action_that_script_is_carrying_out.py    # logic, email sending, api calls, other testable stuff - same name, so we can match up the top level script with its logic
dmscripts/helpers/thing_helpers.py   # any shared functions, classes etc that might apply to more than one script
```

I don't have a problem with class/method based scripts per se, but I'm not sure we should be instantiating it  here in the top level script. Also it feels like for `SupplierFrameworkMethods`, there's some crossover with `dmscripts/helpers/supplier_data_helpers.py` (perhaps we can extend the `SupplierFrameworkData` helper class instead?) and that the call to `supplier_frameworks.remove_declaration_from_suppliers(cutoff_date)` should go in `dmscripts` so it's testable. This is a bit of a rambly comment - happy to chat offline?